app/templates/components/full-main-page-content.hbs

```diff
@@ -0,0 +1,11 @@
+<h2 section="1" {{action 'openSection'}} class="{{if openSection 'open-section'}}">
```

Comment:
do we have to do some shenanigans like here https://github.com/Wikia/mobile-wiki/blob/dev/app/components/article-content.js#L516
when opening this section?

Reply:
You mean aria-shenanigans?

Reply:
nope. gibberish.
article-content is actually rendered below a whole... 
don't listen to me ;)